imap_processing/cli.py

```diff
@@ -618,24 +626,31 @@ def do_processing(self, dependencies):
         elif self.data_level == "l1b":
             data_dict = {}
             for dependency in dependencies:
-                dataset = load_cdf(dependency, to_datetime=True)
+                dataset = load_cdf(dependency, to_datetime=True)  # type: ignore[arg-type]
+                # Todo Change
+                # Argument "to_datetime" to "load_cdf" has incompatible
```

Comment:
Related to this: https://github.com/python/mypy/issues/1969

Reply:
I am a little confused as to how to go about solving this issue.

Reply:
Also, read about kwargs before attempting to understand what I'm saying here: https://realpython.com/python-kwargs-and-args/